web/src/js/components/Modes/Upstream.tsx

```diff
@@ -55,7 +72,7 @@ export default function ReverseToggleRow({
                     dispatch(setActive({ server, value: !server.active }));
                 }}
             >
-                Forward
+                Transfer
```

Comment:
Diff is borked, but I'll comment here anyhow: I think we should move the wording closer to what we have for the explicit HTTP(S) Proxy. How about something like this?


--------------

### Explicit HTTP(S) Proxy (With Upstream Proxy)
All requests are forwarded to a second HTTP(S) proxy server.

- [x] Run HTTP/S Proxy and forward requests to [text input with placeholder: http://example.com:8080].

---------------------------------

I'd be fine with both a single text input for the destination, as well as dropdown + hostname/port inputs. I think for this one it's way more intuitive than for reverse proxy mode, where we "abuse" the scheme a bit with non-standard values such as `udp://`.

Reply:
okay, let's keep the things simple and use just a single input field for the destination :)